src/ng/directive/ngShowHide.js

```diff
@@ -98,7 +98,7 @@ var NG_HIDE_IN_PROGRESS_CLASS = 'ng-hide-animate';
  * @example
   <example module="ngAnimate" deps="angular-animate.js" animations="true">
     <file name="index.html">
-      Click me: <input type="checkbox" ng-model="checked"><br/>
+      Click me: <input type="checkbox" ng-model="checked" aria-label="toggle ngHide and ngShow"><br/>
```

Comment:
Toggling `.ng-hide` is an implementation detail that is of no interest to the user.
Maybe "toggle thumbs-up thumbs-down icons" ?

<sub>(Again, this is not a real app, so it isn't all that significant. The important thing here is to demonstrate proper use of aria attributes.)</sub>